### PR TITLE
gym_location bug fix

### PR DESCRIPTION
**Pull request by williambaibai**
fixed bug for gym_location

### CODE REVIEW

**Comment by young-k** [comment]
Were you able to test this?

File: src/scripts/setup_db.py [hunk 1]
```diff
@@ -243,39 +243,39 @@ def init_data():
     print 'Adding gym_locations to db...'
     # appel_fitness_center
     appel_fitness_center = gd.get_gym_by_id(1)
```
Comment (young-k) [inline src/scripts/setup_db.py]: this shouldn't be hardcoded either, is there a different way we could get these gyms?
i understand it's pretty tricky, but we need a better system of doing this

**Comment by young-k** [comment]
were you able to test this?

**Comment by williambaibai** [comment]
Yes! I tested this and its works as expected.

**Pull request merged**